### PR TITLE
validators directory: Add WeStaking.md

### PR DESCRIPTION
**Pull request by westaking**

### CODE REVIEW

File: WeStaking.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - Telegram: https://t.me/westaking
+    - Discord: westaking
+
+7) Unique validator key to submit (from Cosmos Hub)
+
+    - cosmos validator address : cosmosvaloper1ptyzewnns2kn37ewtmv6ppsvhdnmeapvtfc9y5
+    - govgen validator address : govgenvaloper100kwgdyssd08nuz24guznr9eyy2p8enjjpfq48
+
+8) Region you and/or your validator is based (country)
+    - TODO
```
Comment (albttx) [inline WeStaking.md:36]: We are waiting for full completion to merge

**Comment by VT-Cosmos** [comment]
Thank you for your submission.
@westaking 
There were some updated point that are missing from the application.

Unique validator key to submit (from Cosmos Hub)
Region you and/or your validator is based (country)
We suggest to check the updated requirements to ensure all is covered in the submission, before the deadline.

**Comment by westaking** [comment]
> Thank you for your submission. @westaking There were some updated point that are missing from the application.
> 
> Unique validator key to submit (from Cosmos Hub) Region you and/or your validator is based (country) We suggest to check the updated requirements to ensure all is covered in the submission, before the deadline.

**Comment by westaking** [comment]
Missing items have been added.

**Pull request merged**